core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -59,7 +59,12 @@ class ActivationThrottler(consulServer: String, concurrencyLimit: Int)(
     def check(subject: Subject): Boolean = userActivationCounter.getOrElse(subject(), 0L) < concurrencyLimit
 
     /**
-     * Returns the per-namespace activation count as seen by the loadbalancer
+     * Checks whether the system is in a generally overloaded state.
+     */
+    def isOverloaded = userActivationCounter.values.sum > 5000
```

Comment:
We should move this to ansible config - since its deployment sensitive and we've consolidated the other throttle related params.